src/customer-service/Actions/CreateIncidentAction.cs

```diff
@@ -0,0 +1,158 @@
+                    break;
+                }
+
+                if (orderList.TrueForAll(o => incident.OrderId != orderID) || orderID == 0)
+                {
+                    Console.WriteLine("This is not a valid answer. Try again!");
+                    try
+                    {
+                        orderID = Convert.ToInt32(Console.ReadLine());
+
+                    }
+                    catch
+                    {
+                        Console.WriteLine("Please enter a number");
+                    }
+                }
+            }
+            incident.OrderId = orderID;
+
+          
+            Console.WriteLine("Choose incident type: ");
+            foreach(IncidentType incidentType in incidentTypeList)
+            {
+                Console.WriteLine($"{incidentType.IncidentTypeId} {incidentType.Label}");
+            }
+            Console.WriteLine("X.Exit");
+            string incidentTypeIdAnswer = Console.ReadLine();
+            if (incidentTypeIdAnswer.ToLower() == "x")
+            {
+                // to main menu
+                return;
+            }
+
+            int incidentTypeId = Convert.ToInt32(incidentTypeIdAnswer);
+
+            while (incident.IncidentTypeId <= 0 || incidentTypeList.TrueForAll(o => incident.IncidentTypeId != incidentTypeId))
+            {
+                if (incidentTypeList.Find(it => it.IncidentTypeId == incidentTypeId) != null)
+                {
+                    break;
+                }
+
+                if (incidentTypeList.TrueForAll(o => incident.IncidentTypeId != incidentTypeId) || incidentTypeId == 0)
+                {
+                    Console.WriteLine("This is not a valid answer. Try again!");
+                    try
+                    {
+                        incidentTypeId = Convert.ToInt32(Console.ReadLine());
+
+                    }
+                    catch
+                    {
+                        Console.WriteLine("Please enter a number");
+                    }
+                }
+            }
+
+            while (incident.Resolution == null)
+            {
+                string answer = "";
+                while (answer != "y" || answer != "n")
+                {
+                    Console.WriteLine("Would you like to resolve this issue ? (y / n)");
+                    answer = Console.ReadLine().ToLower();
+                    if (answer == "y" || answer == "n")
+                    {
+                        break;
+                    }
+                }
+                if (answer == "y")
+                {
+                    Console.WriteLine("How was the incident resolved? ");
+                    incident.Resolution = Console.ReadLine();
+                    incident.DateResolved = DateTime.Today;
+                }
+                break;
+            }
+
+            incident.save();
+            Console.WriteLine($"Incident # {incidents.Count } has been added. Press any key to return to main menu.");

```

Comment:
this should be the count + 1, like an auto-incremented value. Basically when I add Incident #5, this WriteLine tells me I added Incident #4